### ISSUE TITLE
Love2d fails to run

### ISSUE THREAD

**Issue by JakobDev** [haikuports/haikuports#5689]
Love2d can't be run
```
~> love
runtime_loader: /boot/system/lib/liblove-11.2.so: Could not resolve symbol 'connect'
resolve symbol "connect" returned: -2147478780
runtime_loader: /boot/system/lib/liblove-11.2.so: Troubles relocating: Symbol not found
~> 
```

**Comment by extrowerk** [comment]
Probably LDFLAGS=“-lnetwork” would be needed, or something like this.

**Comment by korli** [comment]
Should be fixed in 765c24e16f179dcb8af159da7d6e5f49ac6df7c2
